bin/taxref_reformat_gtdb.sh

```diff
@@ -10,7 +10,7 @@ for f in *.tar.gz; do
 done
 
 # Write the assignTaxonomy() fasta file: gtdb_assignTaxonomy.fna
```

Comment:
Not really important, but shouldn't it be just "assignTaxonomy.fna"? Without the "gtdb_".

Reply:
Correct, although the output file has the right name.